src/Fixer/Alias/MbStrFunctionsFixer.php

```diff
@@ -85,8 +86,15 @@ public function isCandidate(Tokens $tokens)
      */
     protected function applyFix(\SplFileInfo $file, Tokens $tokens)
     {
+        $functions = array_filter(
+            self::$functionsMap,
+            static function ($mapping) {
+                return \function_exists($mapping['alternativeName']);
```

Comment:
this was causing the issue. when we execute this, it's always truth, as we have the mb polyfill on execution time...

Reply:
ref #4529

Reply:
cc @SpacePossum